assets/js/components/PostSearcher.js

```diff
@@ -42,7 +42,7 @@ const { useSelect, useDispatch } = Data;
 function PostSearcher() {
 	const [ canSubmit, setCanSubmit ] = useState( false );
 	const [ match, setMatch ] = useState( {} );
-	const analyticsModuleActive = useSelect( ( select ) => select( CORE_MODULES ).isModuleActive( 'analytics' ) );
+	const analyticsModuleConnected = useSelect( ( select ) => select( CORE_MODULES ).isModuleConnected( 'analytics' ) );
```

Comment:
This is weird because it's already in `develop` but shows as a change... 🤔 
https://github.com/google/site-kit-wp/blob/01e0edf785ba0ab381c212359d8da885be1e6620/assets/js/components/PostSearcher.js#L45-L63